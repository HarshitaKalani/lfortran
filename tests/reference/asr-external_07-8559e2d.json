{
    "basename": "asr-external_07-8559e2d",
    "cmd": "lfortran --show-asr --implicit-typing --implicit-interface --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/external_07.f90",
    "infile_hash": "cdc80d386d6ae6fc05a3bbe7c7afa02c47d7f2bf9c226fdb2e9b1e5f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-external_07-8559e2d.stdout",
    "stdout_hash": "aded1dd0df8ca5d06b50e945fef4a3793e1a1c393b6f4be3fb83a7cb",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}