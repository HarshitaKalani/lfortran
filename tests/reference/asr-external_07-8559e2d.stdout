(TranslationUnit
    (SymbolTable
        1
        {
            dqc25f:
                (Function
                    (SymbolTable
                        2
                        {
                            dqk15w:
                                (Function
                                    (SymbolTable
                                        5
                                        {
                                            
                                        })
                                    dqk15w
                                    (FunctionType
                                        [(FunctionType
                                            []
                                            (Real 4)
                                            BindC
                                            Implementation
                                            ()
                                            .false.
                                            .false.
                                            .false.
                                            .false.
                                            .false.
                                            []
                                            []
                                            .false.
                                        )]
                                        ()
                                        BindC
                                        Interface
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        []
                                        .false.
                                    )
                                    []
                                    [(Var 2 f)]
                                    []
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                ),
                            f:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            f_return_var_name:
                                                (Variable
                                                    3
                                                    f_return_var_name
                                                    []
                                                    ReturnVar
                                                    ()
                                                    ()
                                                    Default
                                                    (Real 4)
                                                    ()
                                                    BindC
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    f
                                    (FunctionType
                                        []
                                        (Real 4)
                                        BindC
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        []
                                        .false.
                                    )
                                    []
                                    []
                                    []
                                    (Var 3 f_return_var_name)
                                    Public
                                    .false.
                                    .false.
                                    ()
                                )
                        })
                    dqc25f
                    (FunctionType
                        []
                        ()
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    [dqk15w]
                    []
                    [(SubroutineCall
                        2 dqk15w
                        ()
                        [((Var 2 f))]
                        ()
                    )]
                    ()
                    Public
                    .false.
                    .false.
                    ()
                ),
            external_07:
                (Program
                    (SymbolTable
                        4
                        {
                            
                        })
                    external_07
                    []
                    [(SubroutineCall
                        1 dqc25f
                        ()
                        []
                        ()
                    )]
                )
        })
    []
)
